mlflow/protos/unity_catalog_oss_service.proto

```diff
@@ -26,4 +26,95 @@ service UnityCatalogService {
       visibility: PUBLIC,
     };
   }
+
+  rpc updateRegisteredModel(UpdateRegisteredModel) returns (UpdateRegisteredModel.Response) {
+    option (rpc) = {
+      endpoints: {
+        method: "PATCH",
+        path: "/unity-catalog/models/{full_name}",
+        since { major: 2, minor: 1 },
+      },
+      visibility: PUBLIC_UNDOCUMENTED,
+    };
+  }
+
+  rpc deleteRegisteredModel(DeleteRegisteredModel) returns (DeleteRegisteredModel.Response) {
+    option (rpc) = {
+      endpoints: {
+        method: "DELETE",
+        path: "/unity-catalog/models/{full_name}",
+        since { major: 2, minor: 1 },
+      },
+      visibility: PUBLIC_UNDOCUMENTED,
+    };
+  }
+
+  rpc getRegisteredModel(GetRegisteredModel) returns (GetRegisteredModel.Response) {
+    option idempotency_level = IDEMPOTENT;
+    option (rpc) = {
+      endpoints: {
+        method: "GET",
+        path: "/unity-catalog/models/{full_name}",
+        since { major: 2, minor: 1 },
+      },
+      visibility: PUBLIC_UNDOCUMENTED,
+    };
+  }
+
+  rpc getModelVersion(GetModelVersion) returns (GetModelVersion.Response) {
+    option idempotency_level = IDEMPOTENT;
+    option (rpc) = {
+      endpoints: {
+        method: "GET",
+        path: "/unity-catalog/models/{full_name}/versions/{version}",
+        since { major: 2, minor: 1 },
+      },
+      visibility: PUBLIC_UNDOCUMENTED,
+    };
+  }
+
+  rpc deleteModelVersion(DeleteModelVersion) returns (DeleteModelVersion.Response) {
+    option (rpc) = {
+      endpoints: {
+        method: "DELETE",
+        path: "/unity-catalog/models/{full_name}/versions/{version}",
+        since { major: 2, minor: 1 },
+      },
+      visibility: PUBLIC_UNDOCUMENTED,
+    };
+  }
+
+  rpc createModelVersion(CreateModelVersion) returns (CreateModelVersion.Response) {
+    option (rpc) = {
+      endpoints: {
+        method: "POST",
+        path: "/unity-catalog/models/versions",
+        since { major: 2, minor: 1 },
+      },
+      visibility: PUBLIC_UNDOCUMENTED,
+    };
+  }
+
+  rpc finalizeModelVersion(FinalizeModelVersion) returns (FinalizeModelVersion.Response) {
+    option (rpc) = {
+      endpoints: {
+        method: "PATCH",
+        path: "/unity-catalog/models/{full_name}/versions/{version}/finalize",
+        since { major: 2, minor: 1 },
+      },
+      visibility: PUBLIC_UNDOCUMENTED,
+    };
+  }
+
+  rpc updateModelVersion(UpdateModelVersion) returns (UpdateModelVersion.Response) {
+    option (rpc) = {
+      endpoints: {
+        method: "PATCH",
+        path: "/unity-catalog/models/{full_name}/versions/{version}",
+        since { major: 2, minor: 1 },
+      },
+      visibility: PUBLIC_UNDOCUMENTED,
+    };
+  }
+
```

Comment:
Did we not add FinalizeModelVersion?

Reply:
https://github.com/mlflow/mlflow/pull/12961/files/ef8c929cf642b47386b202734f88c50478b2f532#diff-205db0fe24bdf8ea02d90ac886076dfe3f50b2309ec62f27ef76189d0b0cab58R98-R107